### ISSUE TITLE
Actions menu icon color lacks sufficient color contrast 

### ISSUE THREAD

**Issue by benhammondmusic** [vega/vega-embed#840]
Using a color picker, the measured color of the <svg> is #DADADA, which when I plugged it into a calculator https://webaim.org/resources/contrastchecker/ gives a contrast ratio of **1.39:1**. "WCAG 2.1 requires a contrast ratio of at least **3:1** for graphics and user interface components"

This is caused by the reduced opacity whenever the menu isn't hovered, resulting in a control that could be difficult for some users to see. Proposal to at least provide darker color for users who request high-contrast mode, and ideally should style in a way that meets minimum color contrast guidelines for interactive graphics
![Screen Shot 2022-01-19 at 9 22 06 AM](https://user-images.githubusercontent.com/41567007/150172042-8baf388d-b259-41d7-92fb-bb3b5a653505.png)

**Comment by domoritz** [comment]
Do we meet the contrast ratio on hover or focus? If not, we should fix that. If yes, we can focus on the high contrast request case.
